Whiskey/Tasks/Get-WhiskeyPowerShellModule.ps1

```diff
@@ -0,0 +1,61 @@
+function Get-WhiskeyPowerShellModule 
+{
+    <#
+    .SYNOPSIS
+    Task to get and install a powershell module that is specified. 

```

Comment:
Change to: `Downloads a PowerShell module.`

Reply:
Done